dashboard/app/controllers/api/v1/pd/workshops_controller.rb

```diff
@@ -197,15 +190,8 @@ def update
   def create
     @workshop.organizer = current_user
     adjust_facilitators
-
-    if params[:pd_workshop][:course] == Pd::Workshop::COURSE_BUILD_YOUR_OWN
```

Comment:
Can you walk through what changed between this logic and what's in `adjust_course_offerings`?

Reply:
Thank you! Much easier to parse now -- looks like a great improvement 😁